src/UITests/GettingStartedPage.cs

```diff
@@ -9,29 +9,33 @@ public class GettingStartedPage : AIWinSession
     {
         [TestMethod]
         [TestCategory("NoStrongName")]
-        public void VerifyGettingStartedTitle()
+        public void GettingStartedPageTests()
```

Comment:
>GettingStartedPageTests [](start = 20, length = 23)

At first I thought this was a ctor. Can we make the name clearer?

Reply:
Comment added